src/meta/node/src/lib.rs

```diff
@@ -76,7 +76,7 @@ pub struct MetaNodeOpts {
 
     /// Endpoint of the SQL service, make it non-option when SQL service is required.
     #[clap(long, hide = true, env = "RW_SQL_ENDPOINT")]
-    pub sql_endpoint: Option<String>,
+    pub sql_endpoint: Option<Secret<String>>,
```

Comment:
It looks a bit weird to me to use a URL instead of separated fields (host/ip/username/password). But acceptable.

Reply:
The formats of different endpoints are not quite the same, such as `Sqlite`, and the in-memory mode of `Sqlite` used for testing is also different.

Reply:
A URL like this seems to imply that *any* SQL database can be used as meta backend. Not good to me.